Dockerfile

```diff
@@ -1,22 +1,46 @@
-FROM jmfirth/webpack
-
-# Install packages
-# RUN npm install -g angular-cli && apt-get update && apt-get install git -y;
+FROM node:8.4.0-alpine
 
 # Create Application Directory
+# ENV SASS_BINARY_NAME /usr/share/unfetter-ui/node_modules/node-sass
 ENV WORKING_DIRECTORY /usr/share/unfetter-ui
+# ENV PATH $WORKING_DIRECTORY/node_modules/.bin:$PATH
 RUN mkdir -p $WORKING_DIRECTORY
 WORKDIR $WORKING_DIRECTORY
 
 # Install Dependencies
 COPY package.json $WORKING_DIRECTORY
 
-# The NPM package depends on TAR package, which has a test directory with an encrypted tgz file, that gets blocked by some antivirus scanners. Removing it.
-RUN npm install; find / -name "cb-never*.tgz" -delete; npm cache clean
+RUN apk update && \
```

Comment:
@infosec-alchmeist, thanks for pointing that out, I wasn't sure of its history, but did read the message.  I have the tar removal at the end of the Dockerfile,  see line 34.  Is that good enough?  Or does it need to be specifically at that level in the Docker file cache layer?